src/ray/raylet/node_manager.cc

```diff
@@ -496,8 +496,10 @@ void NodeManager::Heartbeat() {
 }
 
 void NodeManager::DoLocalGC() {
-  auto all_workers = worker_pool_.GetAllRegisteredWorkers();
-  for (const auto &driver : worker_pool_.GetAllRegisteredDrivers()) {
+  auto all_workers =
+      worker_pool_.GetAllRegisteredWorkers(false /* filter_dead_workers */);
```

Comment:
How about set a default value of filter_dead_workers? The previous implementation default is false.

Reply:
Ah yeah. I tried, and it seemed to have a compiler error. But I think I just did it wrong. Is this the right way to do that? 

https://www.programiz.com/cpp-programming/default-argument

Reply:
Yeah, this is the right way.